worktrolly-ui/src/app/body/board/sprint-details/sprint-details.component.ts

```diff
@@ -34,14 +35,32 @@ export class SprintDetailsComponent implements OnInit {
   async completeSprint() {
     
     this.sprintStatus = "Completed";
+    this.enableLoader = true;
     const callable = this.functions.httpsCallable('updateSprintStatus');
 
     try {
       const result = await callable({ CurrentSprintName: this.currentSprintName, SprintStatus: this.sprintStatus }).toPromise();
       console.log(this.sprintStatus);
       console.log("Successfully updated Status");
     } catch (error) {
-      console.error("Error", error);
+    this.enableLoader = false;
+    console.error("Error", error);
+    }
+  }
+
+  async updateSprintStatus() {
+    
+    this.sprintStatus = "Under progress";
+    this.enableLoader = true;
+    const callable = this.functions.httpsCallable('updateSprintStatus');
+
+    try {
+      const result = await callable({ CurrentSprintName: this.currentSprintName, SprintStatus: this.sprintStatus }).toPromise();
+      console.log(this.sprintStatus);
+      console.log("Successfully updated Status");
+    } catch (error) {
+    this.enableLoader = false;
+    console.error("Error", error);
```

Comment:
Reduce the code by reusing the methods with different arguments. So underProgress and completed can use the same method.